src/dispatcher.ts

```diff
@@ -237,7 +237,12 @@ export class Dispatcher {
   }
 
   async initialize(): Promise<void> {
+    this.default.frame.begin();
     await Promise.all(this.systems.map(system => system.initialize()));
+    this.flush();
```

Comment:
Flushing seems to be necessary before calling the `.frame.end()` call, otherwise it fails with:

```
Internal error, should commit log before counting

      234 |   countSince(startPointer: LogPointer, endPointer?: LogPointer): number {
      235 |     CHECK: this.checkPointers(startPointer, endPointer);
    > 236 |     DEBUG: if (this.corral[0]) throw new Error(`Internal error, should commit log before counting`);
          |                                      ^
      237 |     const startIndex = startPointer.index;
      238 |     const startGeneration = startPointer.generation;
      239 |     const endIndex = endPointer?.index ?? this.data[0];
```